data/severity.go

```diff
@@ -0,0 +1,6 @@
+package data
+
+// Severities consists of severity levels
+var Severities = map[string][]string{
```

Comment:
This map should not be exported.
It is mutable and the only access methods are enough.

Also, I'm wondering if we need a map at all for this.

A simple slice of `LogLevel` type (see my comment below) should suffice here.

Reply:
I think you could have the severity levels straight in the function you create for this.

```go
func Severity () string {
    // Randomly pick one 
    return value
}
```

Reply:
Will Go compiler optimize that to avoid putting those values on the function call stack every time?
There is an escape analysis done by the compiler but I don't know about this case.

Reply:
I see what your saying. you could just do a private var right above that function if thats the case

Reply:
I could go either way

Reply:
@hbagdi Level is for full, LevelCode is for short?